src/main/java/com/yilmaznaslan/errorhandling/FormValidationFailedException.java

```diff
@@ -1,7 +1,9 @@
 package com.yilmaznaslan.errorhandling;
 
 public class FormValidationFailedException extends Exception {
-    public FormValidationFailedException(String message) {
+    private static final long serialVersionUID = 1L;
```

Comment:
Why is it needed

Reply:
To suppress the following Eclipse warning:
"The serializable class FormValidationFailedException does not declare a static final serialVersionUID field of type long"

Reply:
I ' suggest  that we shouldn't add functionality or complexity until it's actually needed. 

Given that we don't currently face serialization-related problems, this overhead doesn't seem to be necessary. You can ignore the warning ;). I highly suggest to you use sonarlint plugin btw instead custom eclipse code analyzer

Reply:
I see (and agree on) your point in avoiding over-engineering.

That being said, I don't see how adding *one line* that gets rid of a compiler warning can be a case of over-engineering - specially considering that the workaround implies either using an IDE-specific plugin or adding another line of code (in this case, an annotation)...

Reply:
My comment above addresses https://github.com/yilmaznaslan/berlin-auslaenderbehorde-termin-bot/pull/75#pullrequestreview-1677812358